app/graphql/mutations/update_user.rb

```diff
@@ -5,6 +5,7 @@ class UpdateUser < BaseMutation
     argument :last_name, String, required: false
     argument :current_password, String, required: false
     argument :password, String, required: false
+    argument :avatar, String, required: false
```

Comment:
```suggestion
    argument :avatar, Types::ImageUploaderType, required: false
```

I think that we should use the same approach here.